lib/bibliothecary/parsers/generic.rb

```diff
@@ -24,7 +24,7 @@ def self.parse_lockfile(file_contents)
         table.map.with_index do |row, idx|
           line = idx + 1
           required_headers.each do |h|
-            raise "missing field '#{h}' on line #{line}" if row[h].empty?
+            raise "missing field '#{h}' on line #{line}" if row[h].nil? || row[h].empty?
```

Comment:
Ruby noob: If row[h] was `nil` wouldn't we have seen an `undefined method `empty?' for nil:NilClass` instead of passing? (I was trying to see if `nil` was empty locally).

(Side bar I think returning nil for an `,,` instead of an empty string is dumb, but that's the CSV lib, not us)

Reply:
Oh! I couldn't find the error anywhere when I was looking.

Reply:
Oh wait. I was looking in depci logs. Maybe that was the wrong spot.